src/20_Components/amp-fx-flying-carpet.html

```diff
@@ -0,0 +1,63 @@
+<!---{
+    "experiment": true,
+    "component": "amp-fx-flying-carpet"
+  }--->
+<!--
+  #### Introduction
+
+  [amp-fx-flying-carpet](https://github.com/ampproject/amphtml/blob/master/extensions/amp-fx-flying-carpet/amp-fx-flying-carpet.md) displays its children inside a container of fixed height. As the user scrolls the page, the flying carpet reveals more of it contents, sliding across its children as if peering through a window in the page.
+
+  For best experience of this example, use a mobile device or choose a mobile device layout using the Chrome console.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!-- Import the `amp-fx-flying-carpet` component in the header. -->
+  <script async custom-element="amp-fx-flying-carpet" src="https://cdn.ampproject.org/v0/amp-fx-flying-carpet-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!--
+    `amp-fx-flying-carpet` can be used to display images.
+    Use height parameter to specify the height of the flying carpets "window".
+
+    Use vw/vh to size elements inside the `amp-fx-flying-carpet` to make their size relative to the size of the viewport. This will allow the best scrolling effect having the element using the entire viewport. 
+  -->
+  <amp-fx-flying-carpet height="200px">
+     <amp-img src="/img/landscape1.jpg"
+      width="100vw"
+      height="100vh"
+      alt="an image"></amp-img>
+  </amp-fx-flying-carpet>
+ 
+     <!--
+     #### Positioning
+     -->
+     <!--
+
+    `amp-fx-flying-carpet` elements must be positioned before the last viewport
```

Comment:
Does it need to be placed after the first view port as well? Can we specify better what "some content after the component" means? maybe: "so make sure to place enough content after the element to cover the targeted viewport sizes."